lib/cog_api/http/relay_groups.ex

```diff
@@ -38,30 +38,19 @@ defmodule CogApi.HTTP.RelayGroups do
     |> ApiResponse.format_delete("The relay group could not be deleted")
   end
 
-  def update_memberships(action, %{name: name}, %{relay: relay_name}, %Endpoint{}=endpoint) do
-    case get_group_relay(name, relay_name, endpoint) do
-      {:ok, relay, relay_group} ->
-        update_membership(relay_group.id, relay.id, action, endpoint)
-      error ->
-        error
+  def update_memberships(action, %{name: name}, %{relays: relay_names}, endpoint) when is_list(relay_names) do
```

Comment:
Is this function ever publicly called with anything other than names? The maps-as-a-differentiator here make for confusing-to-read code.